dspace-server-webapp/src/main/java/org/dspace/app/rest/utils/RegexUtils.java

```diff
@@ -22,7 +22,7 @@ private RegexUtils(){}
      * identifier (digits or uuid)
      */
     public static final String REGEX_REQUESTMAPPING_IDENTIFIER_AS_STRING_VERSION_STRONG = "/{id:^(?!^\\d+$)" +
-        "(?!^[0-9a-fxA-FX]{8}-[0-9a-fxA-FX]{4}-[0-9a-fxA-FX]{4}-[0-9a-fxA-FX]{4}-[0-9a-fxA-FX]{12}$)[\\w+\\-]+$+}";
+        "(?!^[0-9a-fxA-FX]{8}-[0-9a-fxA-FX]{4}-[0-9a-fxA-FX]{4}-[0-9a-fxA-FX]{4}-[0-9a-fxA-FX]{12}$)[\\w+\\-\\.]+$+}";
```

Comment:
I don't quite understand this change.  Should we update the Javadocs description of this Regex to describe the format it expects?  Or maybe give an example or two.

Reply:
well I don't know really how to improve the javadoc here. It should be everything that is not an UUID or a digit. My fix is to be sure that identifier containing a dot (".") are allowed as for instance
`8c7b9132-eadc-4199-af6d-3260a678e96f_withdrawItem_core.item_c8924526-67ef-479a-8e37-dd8d19e625e9` that by accident can be the identifier of an authorization

Reply:
Ok, this makes sense to me & seems reasonable. Prior to enhanced JavaDocs, I wasn't aware that these identifiers could have dots/periods.